drivers/gpio/Kconfig.pcf857x

```diff
@@ -0,0 +1,20 @@
+# PCF857x GPIO configuration options
```

Comment:
Not sure why this one is not detected as a rename

Reply:
Is this still a blocker?

Reply:
guess not